sources/packages/backend/apps/api/src/route-controllers/user/user.aest.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { Controller, Get, Put } from "@nestjs/common";
+import { UserService } from "../../services";
+import BaseController from "../BaseController";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import { BCeIDAccountsAPIOutDTO } from "./models/bceid-accounts.dto";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { ApiTags, ApiUnprocessableEntityResponse } from "@nestjs/swagger";
+import { UserControllerService } from "..";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Controller("users")
+@ApiTags("users")
+export class UserAESTController extends BaseController {
+  constructor(
+    private readonly service: UserService,
+    private readonly userControllerService: UserControllerService,
+  ) {
+    super();
+  }
+
+  @ApiUnprocessableEntityResponse({
```

Comment:
Please add some comments, even basic ones, to keep the culture of trying to comment all the methods as much as possible. This comment also goes to swagger.